### PR TITLE
Granular rename of the openapi models

### PR DESCRIPTION
**Pull request by wtrocki**
## Description

Applying suggestions from #404. 
I realized that PR build did not include files that were deleted.  PR removes unneeded abandoned files. 
SDK still have a lot of unused models but that is out of the scope of this PR.

### CODE REVIEW

File: api/test/regression/database_users.go [hunk 1]
```diff
@@ -5,7 +5,7 @@ import (
 )
 
 func TestDatabaseUsersModelRegression() {
-	_ = mongodbatlas.PaginatedApiAtlasDatabaseUserView{
```
Comment (wtrocki) [inline api/test/regression/database_users.go:8]: Leftover from the previous PR. Renamse for models now failed build due to models were still present.

File: tools/transformer/src/transform.js [hunk 2]
```diff
@@ -19,9 +19,9 @@ doc = applyDiscriminatorTransformations(doc);
 doc = applyOneOfTransformations(doc);
 doc = applyAllOfTransformations(doc);
 
-doc = applyModelNameTransformations(doc, "ApiAtlas", "View", ignoredModelNames);
-doc = applyModelNameTransformations(doc, "Api", "View", ignoredModelNames);
-doc = applyModelNameTransformations(doc, "", "View", ignoredModelNames);
 doc = applyModelNameTransformations(doc, "Api", "", ignoredModelNames);
+doc = applyModelNameTransformations(doc, "Atlas", "", ignoredModelNames);
```
Comment (wtrocki) [inline tools/transformer/src/transform.js:23]: Core of the change. Basically apart from the Manual views and ApiUsers there are no extra objects renamed.

File: api/v1alpha/README.md [hunk 3]
```diff
@@ -346,8 +346,10 @@ Class | Method | HTTP request | Description
  - [AlertConfigAuditType](docs/AlertConfigAuditType.md)
```
Comment (wtrocki) [inline api/v1alpha/README.md:346]: List of changed models

File: api/v1alpha/README.md [hunk 4]
```diff
@@ -364,13 +366,6 @@ Class | Method | HTTP request | Description
  - [AppServiceMetricMatcherField](docs/AppServiceMetricMatcherField.md)
  - [AppServiceMetricThreshold](docs/AppServiceMetricThreshold.md)
  - [AppUser](docs/AppUser.md)
- - [AtlasContainerPeerViewRequest](docs/AtlasContainerPeerViewRequest.md)
```
Comment (wtrocki) [inline api/v1alpha/README.md:367]: This is extra case - Some of the objects had prefix Atlas but not ApiAtlas

**Review by gssbzn — APPROVED** [review]
LGTM

File: tools/transformer/src/transform.js [hunk 5]
```diff
@@ -19,9 +19,9 @@ doc = applyDiscriminatorTransformations(doc);
...
+doc = applyModelNameTransformations(doc, "", "View", ignoredModelNames);
```
Comment (andmatei) [inline tools/transformer/src/transform.js:24]: I think these should be inverted, as manual views end in "ViewManual" - so we should delete "View" first, then "Manual".

**Review by andmatei — CHANGES_REQUESTED** [review]
Only one concern, and then LGTM

**Review by andmatei — APPROVED** [review]
LGTM

**Pull request merged**